bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/NetatmoBindingConstants.java

```diff
@@ -58,6 +58,7 @@ public class NetatmoBindingConstants {
     public static final String GROUP_PROPERTIES = "properties";
     public static final String GROUP_SETPOINT = "setpoint";
     public static final String GROUP_LOCATION = "location";
+    public static final String GROUP_BRIDGE = "bridge";
```

Comment:
As mentioned in another comment, "monitoring" or something like that could be more appropriate.